.github/actions/tests/run-e2e-tests/action.yml

```diff
@@ -95,22 +95,28 @@ runs:
           echo "Running tests via docker exec"
           docker exec tests bash -c "
             cd /e2e-tests &&
+            python -m venv venv &&
             source venv/bin/activate &&
+            pip install -r requirements.txt &&
             $pytest_cmd
           "
         else
           echo "Running tests locally"
           cd e2e-tests
+          python -m venv venv
           source venv/bin/activate
+          pip install -r requirements.txt
```

Comment:
I think the new lines are redundant, it should be handled in `dev/local-environment/configurations/tests/entrypoint.sh` already